.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+## Description
```

Comment:
you can remove this file, it will come with the sync-files action

Reply:
I just added just in case so that we have a template when someone creates a pull request before sync-file PR is merged, but it shouldn't be a critical issue so I just removed the file.